src/news/2021-07-13-Web-Site-Starter-Kits.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Web Site Starter Kits
+filename: Web-Site-Starter-Kits
+date: '2021-07-13'
+---
+The FLOE Team is developing starter kits to accelerate the development of web sites; providing a solid foundation to
+build from and maintain over time.
+
+[Trivet](https://github.com/fluid-project/trivet) is a simple starter kit for using [Eleventy](https://www.11ty.dev) to
+create accessible, localizable static sites.
+
+[Hearth](https://github.com/fluid-project/hearth) was created to support the creation of and access to libraries of open
+educational resources. Hearth is a starter kit for the [Laravel](https://laravel.com) framework, providing the following
+out of the box:
+
+* A user model with login, registration, email verification, and optional two-factor authentication.
+* An organization model.
+* A membership model which reflects users' roles within organizations.
+* An invitation model which allows users to be invited to join organizations.
+* A resource model supporting creation of and access to a library of open educational resources in a wide range of formats.
+* Multilingual support, including localized routes.
+* Bare bones views which can be published and customized as needed.
```

Comment:
Might be worth adding examples of sites that have been built using Hearth and Trivet if they exist so people can see how it has been used.

Reply:
@greatislander and @waharnum, has anything been built with Hearth yet?